### DISCUSSION TITLE
提高兼容性：添加一个 gpt-3.5-turbo 作为 gemma-7b-it（或其他模型）的外观

### DISCUSSION THREAD

**Discussion by JoeyLearnsToCode** [Star-Studio-Develop/Groq2API#32] [Ideas]
很多使用 api 的场景对自定义模型的支持不够好，比如大热的 chrome 拓展 [sider](https://chrome.google.com/webstore/detail/difoiogjjojoaoomphldepapgpbgkhkb)。
可以添加一个假的 gpt-3.5-turbo 模型，它是 gemma-7b-it（或其他模型）的外观，在收到 model 为 gpt-3.5-turbo 的对话请求时，将其替换成实际的模型即可。
这样可以提高使用场景的兼容性。
我已经简单实现：https://github.com/JoeyLearnsToCode/Groq2API
不过我不确定是否有通用性，仓库主可以评估下，如果有必要我再提交 pull request

**Comment by xiaozhou26** [comment]
你可以直接pr的
